lib/posix/pthread.c

```diff
@@ -278,11 +278,11 @@ int pthread_setschedparam(pthread_t pthread, int policy,
 int pthread_attr_init(pthread_attr_t *attr)
 {
 
-	if (attr->initialized == true) {
-		return EBUSY;
+	if (attr == NULL) {
+		return ENOMEM;
```

Comment:
I guess I would have expected an `EINVAL` in this case.

Reply:
from spec: 
ENOMEM : Insufficient memory exists to initialize the thread attributes object.

Here i assumed the input attr has insufficient memory.